### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by TadesseDev**
# Changes introduced 😎
---
- Initialize React app ✔
- Create reusable components/ ✔
> `Books`, `Categories`, `Navigation` ✔
> Also `AddNewForm` and single `Book ` 👌🏼
- Add Initial HTML contents.
# Node: 📝
>## Styles are not applied in this pull request. ✌

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "name": "book-store",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.1.1",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
```
Comment (mateo951) [inline package.json:11]: `react-router-dom` is present in `package.json` ✔

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const ulStyle = {
+    display: 'flex',
+    gap: '10px',
+    listStyle: 'none',
+  };
+  return (
+    <nav style={navStyle}>
+      <ul style={ulStyle}>
+        <li><NavLink to="/">Book</NavLink></li>
+        <li><NavLink to="/categories">Category</NavLink></li>
```
Comment (mateo951) [inline src/components/Navbar.js:18]: You have 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🔥

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import Navbar from './components/Navbar';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
```
Comment (mateo951) [inline src/index.js:15]: Has 2 `<Route>`s, one for the BookList component and the other for the Categories component ✔

- [x] BookList component is the default page

File: src/components/Books.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    },
+  ];
+  return (
+    <div>
+      {books.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} />
+      ))}
+      <AddNewBook />
+    </div>
+  );
```
Comment (mateo951) [inline src/components/Books.js:25]: Renders a list of books using the `Book` component 🔥
Renders a form by using the CreateNewBook component 😎

**Review by mateo951 — APPROVED** [review]
### Hi @Tadesse-Alemayehu, 

📝👌 **Your project highlights** 🔥😎 
- General requirements ✔
- Components added flawlessly 💪

Your project is complete!  There is nothing else to say other than... it's time to merge it 🧐
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
### Dear @mateo951 Thank you, it's good to know I did right. 👍

**Pull request merged**